pkg/proxy/iptables/proxier.go

```diff
@@ -1188,30 +1190,38 @@ func (proxier *Proxier) syncProxyRules() {
 				writeLine(proxier.natChains, utiliptables.MakeChainLine(svcXlbChain))
 			}
 			activeNATChains[svcXlbChain] = true
-		} else if activeNATChains[svcXlbChain] {
-			// Cleanup the previously created XLB chain for this service
-			delete(activeNATChains, svcXlbChain)
```

Comment:
This part seems to be "lost", is that intentional and the new structure doesn't need this bit anymore?